### PR TITLE
uplode t6

### PR DESCRIPTION
**Pull request by asifnaseebi**

### CODE REVIEW

File: task/t6.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from utils.reader import file_read
+from datetime import datetime
+file_content = file_read("/home/asif/Desktop/project/barq_weather_task/files/f2.csv")
+
+for file_value in file_content:
+    date = file_value[1]
+    datetime = datetime.strptime(date,"%Y-%m-%d")
+    weekday = datetime.strftime("%A")
+    events = file_value[-2]
```
Comment (Hamma111) [inline task/t6.py:9]: `event` would be a better name since it is only one event

File: task/t6.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from utils.reader import file_read
+from datetime import datetime
+file_content = file_read("/home/asif/Desktop/project/barq_weather_task/files/f2.csv")
```
Comment (Hamma111) [inline task/t6.py:3]: always 2 new lines space after imports. this mistake should not happen again.

File: task/t6.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+for file_value in file_content:
+    date = file_value[1]
+    datetime = datetime.strptime(date,"%Y-%m-%d")
+    weekday = datetime.strftime("%A")
+    events = file_value[-2]
+    if events == "Thunderstorm":
+        print(weekday + ' = Thunderstorm')
```
Comment (Hamma111) [inline task/t6.py:11]: new line at the end! this should not happen again!

**Review by Hamma111 — CHANGES_REQUESTED** [review]

**Comment by asifnaseebi** [comment]
it was changed again. 22 march

**Pull request merged**